config-connector/solutions/sql/kpt/mysql-ha/README.md

```diff
@@ -0,0 +1,32 @@
+MySQL High Availability
+==================================================
+# NAME
+  mysql-ha
+# SYNOPSIS
+  Config Connector compatible YAMLs for creating a high availability MySQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/config-connector/solutions/sql/kpt/mysql-ha mysql-ha
+  ```
+# USAGE
+## SETTERS
+|     NAME      |       VALUE       |     SET BY      |       DESCRIPTION       | COUNT |
+|---------------|-------------------|-----------------|-------------------------|-------|
+| database-name | mysql-ha-database | package-default | name of SQL database    | 1     |
+| instance-name | mysql-ha-instance | package-default | name of SQL instance    | 5     |
+| password      | ${PASSWORD?}      | PLACEHOLDER     | password of SQL user    | 1     |
```

Comment:
Password from here will make its way to secrets data section, which should be base 64 encoded. We should mention in the description that this is base64 encoded password.